plugins/performance-lab/includes/site-health/avif-headers/helper.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Helper function to detect and test AVIF header information.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for avif_headers test.
+ *
+ * @since n.e.x.t
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function avif_headers_check_avif_headers_test(): array {
+	$result = array(
+		'label'       => __( 'Your site sends AVIF image headers', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+			'<p>%s</p>',
+			esc_html__( 'AVIF images served without the `image/avif` `content-type` header may not render properly in the browser.', 'performance-lab' )
```

Comment:
Since backticks aren't turned in to `code`, maybe they can just be removed?

```suggestion
			esc_html__( 'AVIF images served without the Content-Type of image/avif may not render properly in the browser.', 'performance-lab' )
```

Reply:
This should use `sprintf` for values such as `image/avif` and `Content-Type` (if referring to the header specifically), and have a translator comment.

Reply:
> This should use `sprintf` for values such as `image/avif` and `Content-Type` (if referring to the header specifically), and have a translator comment.

Ah, good point, I will rework it that way.

Reply:
Took a pass at improving this in [`84292b0` (#1612)](https://github.com/WordPress/performance/pull/1612/commits/84292b0a3b2073ed7d686ab978f5d2836c7432ee)